reference/events.v1.yaml

```diff
@@ -73,11 +75,23 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/event'
+      x-tags:
+        - events
+      description: 'A list of events - reverse chronologically ordered, provided to enable subscribers to ''catch up'' on events they may have missed.'
```

Comment:
Is that unlimited? or is that dependent on the service (like: all events from beginning of time, or events from last 2 weeks max)

Reply:
Typically unlimited, but ultimately a provider could choose to add a limit if it makes sense.   We do need to discuss how the first sync happens for a new subscriber, this is something we missed in the first round of talks.